doc/source/install.rst

```diff
@@ -249,10 +249,9 @@ Optional Dependencies
 * `statsmodels <http://statsmodels.sourceforge.net/>`__
    * Needed for parts of :mod:`pandas.stats`
 * `openpyxl <http://packages.python.org/openpyxl/>`__, `xlrd/xlwt <http://www.python-excel.org/>`__
-   * openpyxl version 1.6.1 or higher, but lower than 2.0.0
```

Comment:
Yes - both 1.x and 2.x are clearly supported and tested in excel.py and test_excel.py.  I suppose the person who added the OpenPyXL 2.x path forgot to change this.